chromium/browser_type.go

```diff
@@ -240,22 +239,26 @@ func (b *BrowserType) allocate(
 		return nil, err
 	}
 
-	path := opts.ExecutablePath
-	if path == "" {
-		path = b.ExecutablePath()
+	path, ok := ExecutablePath(opts.ExecutablePath, b.envLookupper, exec.LookPath)
+	if !ok {
+		return nil, ErrChromeNotInstalled
 	}
 
 	return common.NewLocalBrowserProcess(bProcCtx, path, args, dataDir, bProcCtxCancel, logger) //nolint: wrapcheck
 }
 
+// ErrChromeNotInstalled is returned when the Chrome executable is not found.
+var ErrChromeNotInstalled = errors.New("chromium is not installed on this system")
```

Comment:
I've exported this in case we'll want to test against it later on.

Reply:
Maybe it needs to be `chrome/chromium` otherwise people might wonder whether chrome is supported or not. I think i see questions about that in the community sometimes.

Reply:
See 6c46e7c8.